idmanager/idmanager.go

```diff
@@ -62,18 +61,17 @@ func (m *manager) Generate(ctx context.Context) (int, error) {
 		return -1, errors.Wrapf(err, "fail to get list of items with prefix %s from store", m.prefix)
 	}
 
-	var ids []int
+	ids := map[int]bool{}
```

Comment:
Why this change was there an error ?

If yes, could you write a specs for this error ?

Reply:
The behavior was described here: https://github.com/Scalingo/sand/issues/51

I'll add a test about it